RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -562,6 +562,12 @@ boolean auto_pre_adventure()
 		addBonusToMaximize(exting, 200); // extinguisher prevents per-round hot damage in wildfire path 
 	}
 
+	if(!(contains_text(get_property("_perilLocations"), place.id)) && auto_havePeridot() && zoneHasWantedMonsters)
```

Comment:
you'll need to split the string first, contains_text won't work because a location id could be a substring of another location's id.

I would make a helper function in mr2025.ash